torchqdynamics/utils/utils.py

```diff
@@ -22,95 +22,106 @@
 ]
 
 
-def is_ket(x: Tensor) -> Tensor:
-    """Check if a tensor is in state vector format."""
+def is_ket(x: Tensor) -> bool:
+    """Check if a tensor is in state vector format.
+
+    Args:
+        x ( Tensor (..., d, n) ): State vector, linear map or density matrix.
+
+    Returns:
+        `True` if the last dimension `n` is `1`, `False` otherwise.
+    """
     return x.size(-1) == 1
 
 
 def ket_to_bra(x: Tensor) -> Tensor:
-    """Linear map (bra) representation of a state vector (ket).
+    r"""Linear map $\bra\psi$ representation of a state vector $\ket\psi$.
 
     Args:
-        x: Tensor of dimension `(..., d, 1)`.
+        x ( Tensor (..., d, 1) ): State vector.
```

Comment:
```suggestion
        x ( Tensor (..., n, 1) ): State vector.
```
Throughout the documentation as well.